### PR TITLE
Jump through progress bar + Other fixes

### PR DESCRIPTION
**Pull request by tuantinghuang**
The main change is adding in the feature to allow viewers to jump through video timeline with different sections. Also did some other fixes:
* optimized audio
* added in transcript
* ending call-to-action frame (Donate box styling will be investigated by Xin)
* redesigned beginning frame

The video files used in the code now can be downloaded [here](https://drive.google.com/drive/folders/1D6-TqcziiBBpcAsgFhI6CLZcZmy6g_iF?usp=drive_link) and run local temporarily. Will transfer to Processing's CDN once the video files are finalized.

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -20,7 +19,7 @@ var main = {
         console.log(script.popcorn.currentTime());
         var clickX = e.pageX - ($(window).width() / 2)
         clickY = $(window).height() - e.pageY;
-        console.log("{left:" + clickX + ", top:" + clickY + "}");
+        console.log("{left:" + e.pageX + ", top:" + clickY + "}");
```
Comment (ksen0) [inline js/main.js:22]: Intended console log?

File: js/script.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ var script = {
   popcorn: null,
   init: function () {
 
-    var pop = Popcorn.smart("#videoClip", ["https://storage.cloud.google.com/hello-p5-bucket/p5-video-draft-3.webm", "https://storage.cloud.google.com/hello-p5-bucket/p5-video-draft-3.mp4"], { width: 1920, height: 1080 });
+    var pop = Popcorn.smart("#videoClip", ["/assets/p5-video-draft-4.webm", "/assets/p5-video-draft-4.mp4"], { width: 1920, height: 1080 });
```
Comment (ksen0) [inline js/script.js:5]: @tuantinghuang The public link is here: https://hello-assets.p5js.org/p5-video-draft-3.webm - please check if this works?

Comment (ksen0) [reply js/script.js:5]: And mp4: https://hello-assets.p5js.org/p5-video-draft-x264.mp4

File: js/main.js (hunk 1 above)
Comment (tuantinghuang) [reply js/main.js:22]: Sometimes when debugging, we need to fix the CSS with this.

File: js/main.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 var main = {
   sketch: null,
   scaleFactor: 1.0,
-  debug: false,
+  debug: true,
```
[the remark this answers is not in the document]
Comment (tuantinghuang) [reply js/main.js:4]: Yes! Fixing it now

File: js/script.js (hunk 2 above)
Comment (tuantinghuang) [reply js/script.js:5]: Both link works, thank you! I will send the newest version of the video files once I fix more things!

**Pull request merged**